WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/AvatarTrainView.swift

```diff
@@ -0,0 +1,90 @@
+import UIKit
+
+/// A view that shows a train of faces.
```

Comment:
Can this say "avatars" instead of "faces"?

Reply:
Yep! Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/18329/commits/f0297f58687e47deacd07f614a7fac099cbafb47.